Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM php:7.4.19-apache
+FROM php:7.4.20-apache
 LABEL maintainer="matt_henderson@sil.org"
 
 RUN apt-get update -y && \
```

Comment:
I would recommend switching to apt instead of apt-get. The latter is a wrapper and may hide useful messages which could be responded to while building.

Reply:
Good to know about, thanks